### ISSUE TITLE
Report from iframe component not showing details (rel 2.4)

### ISSUE THREAD

**Issue by pcieslak** [galenframework/galen#601]
Before 2.4, all errors from component in iframe were shown on the report.
Currently there is only summary info on issues#...
![image](https://user-images.githubusercontent.com/18451235/49515700-f640eb80-f897-11e8-8a69-1ea51156b22b.png)

Can it be fixed to work as on ver < 2.4 ?

**Comment by ZeleniiZmey** [comment]
Any update on this?

**Comment by ishubin** [comment]
I am currently working on it but it appeared a bit more complicated than I thought. Might take some time since I totally overlooked sublayouts in components specs.

**Comment by ishubin** [comment]
I have released a fix in v 2.4.2. Could you please check it?

**Comment by pcieslak** [comment]
Looks fine - after quick check. If anything will be wrong - will reply by end of this week.

**Comment by pcieslak** [comment]
after some time of usage, I found a case when it is not working. When component partially fails/partially passes. See screenshot
![image](https://user-images.githubusercontent.com/18451235/52216859-82bb5a80-2897-11e9-979b-98b5d2631b76.png)

**Comment by ishubin** [comment]
@pcieslak Strange, looks like the spec "centered all inside asyncButton 12px" is not rendered in the report at all. I will try to reproduce it.

**Comment by pcieslak** [comment]
@ishubin - it looks like it is not showing rules result.

**Comment by ishubin** [comment]
@pcieslak I tried to reproduce but for me it did work correctly

<img width="1107" alt="screen shot 2019-02-11 at 21 13 49" src="https://user-images.githubusercontent.com/949577/52590613-05b55580-2e42-11e9-9bf4-b3def9ac06db.png">

**Comment by ishubin** [comment]
Here is the sample project I used for reproducing the issue
[galen-issue-601.zip](https://github.com/galenframework/galen/files/2852905/galen-issue-601.zip)

**Comment by ZeleniiZmey** [comment]
Ok. Issue is reproducible, when rule failed. I've just updated main.gspec and run your project against it.
```
@objects
    header      h3
    link        a


@rule %{name} should be clickable
    ${name}:
        height 2 to 50px

@rule above link should be %{from} to %{to}
        above link ${from} to ${to}px

= Inside frame test =
    header:
        | above link should be 5 to 10
        height 2 to 45px

    | header should be clickable
```

Here is the output:
![screen](https://user-images.githubusercontent.com/10129917/52626070-b1968980-2eb2-11e9-9b30-c752b38573d8.png)

**Comment by ishubin** [comment]
Thanks for reporting it @ZeleniiZmey I actually missed those completely in the new report implementation. I have pushed and released a fix in 2.4.3

**Comment by ZeleniiZmey** [comment]
It is ok now. Looks like we can close the issue. Many thanks.
